### PR TITLE
Interop - Expand RHSSO on ROSA to 4.17

### PR DESCRIPTION
**Pull request by oharan2**
Passed Job:

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/57809/rehearse-57809-periodic-ci-rhbk-sso-test-main-rhsso-product-ocp4.17-lp-rosa-classic-rhsso-interop-aws/1846085147678478336

(considered as passed since the failure is a temp ROSA deprovision issue)

### CODE REVIEW

File: ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.17-lp-rosa-classic.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      CLO_SUB_CHANNEL: stable-6.0
+      CLO_SUB_SOURCE: redhat-operators
+      E2E_RUN_TAGS: '@rosa|@level0'
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "ipi-install-*", "failure_type": "pod_failure", "classification": "IPI Install RBAC", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "ipi-install-*", "failure_type": "test_failure", "classification": "IPI Install RBAC", "jira_project": "Openshift Logging", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"], "jira_assignee":"anli@redhat.com"},
+              {"step": "openshift-extended-*", "failure_type": "pod_failure", "classification": "Test Failure", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
```
Comment (Simranpal) [inline ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.17-lp-rosa-classic.yaml:68]: Is the jira assigneed needed here as well?

Comment (oharan2) [reply ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.17-lp-rosa-classic.yaml:68]: This comes from the initial scenario onboarded on AWS IPI.
I think that if it was specifically mentioned, it shouldn't be ovierrden, until PQE asks to remove :)

Comment (oharan2) [reply ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.17-lp-rosa-classic.yaml:66]: Taking a second look, the `ipi-install-*` steps could be removed from the firewatch config. Updated in the last commit.

**Pull request merged**